test/cli/premium_test.py

```diff
@@ -26,48 +28,43 @@ def __copy_cppcheck_premium(tmpdir):
                 {
                     "addons": [],
                     "productName": "NAME",
-                    "about": "NAME",
-                    "safety": true
+                    "about": "NAME"
                 }
                 """.replace('NAME', __PRODUCT_NAME))
 
     return exe
 
 
 def test_misra_c_builtin_style_checks(tmpdir):
-    # FIXME this test does not work in ci-windows.yml (release build)
-    if sys.platform == 'win32':
-        return
-
     test_file = os.path.join(tmpdir, 'test.cpp')
     with open(test_file, 'wt') as f:
         f.write('void foo() { int x; y = 0; }')
 
     exe = __copy_cppcheck_premium(tmpdir)
 
-    exitcode, _, stderr = cppcheck(['--premium=autosar', '--xml', test_file], cppcheck_exe=exe)
+    exitcode, _, stderr = cppcheck(['--premium=autosar', '--premium=safety', '--xml', test_file], cppcheck_exe=exe)
     assert exitcode == 0
     assert 'id="unusedVariable"' in stderr
     assert 'id="checkersReport"' in stderr
 
-    exitcode, _, stderr = cppcheck(['--premium=autosar', '--premium=safety-off', '--xml', test_file], cppcheck_exe=exe)
+    exitcode, _, stderr = cppcheck(['--premium=autosar', '--xml', test_file], cppcheck_exe=exe)
     assert exitcode == 0
     assert 'id="unusedVariable"' in stderr
     assert 'id="checkersReport"' not in stderr
```

Comment:
I expect to see checkersReport in the Cppcheck Premium output unless `--premium=safety-off` is provided.

Reply:
Actually I removed it in the changes and inverted the checks. I reverted that change.